tests/ReactTestApp/ios/ReactTestApp.xcodeproj/project.pbxproj

```diff
@@ -565,8 +575,8 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				ALWAYS_SEARCH_USER_PATHS = NO;
-				CC = "$(SRCROOT)/../../../scripts/ccache-clang.sh";
-				CLANG_CXX_LANGUAGE_STANDARD = "c++17";
+				CLANG_ANALYZER_LOCALIZABILITY_NONLOCALIZED = YES;
+				CLANG_CXX_LANGUAGE_STANDARD = "gnu++0x";
```

Comment:
Our c++ is c++17.

Reply:
Am I right in assuming that the underlying Pods would override the compiler here?  Or is some of the testing code also needing C++17?